src/Samples/PortableTask/PortableTask.csproj

```diff
@@ -12,7 +12,11 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Build.Framework" Version="15.5.180" />
-    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.5.180" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.5.180" GeneratePathProperty="true" />
+    <PackageDownload Include="Microsoft.Build.Utilities.Core" Version="[15.5.180]" />
```

Comment:
If you already have a `PackageReference` to this, you shouldn't need the `PackageDownload`.